tests/test_crypto/test_ethereum_crypto.py

```diff
@@ -46,6 +46,6 @@ def test_initialization():
 def test_sign_transaction():
     """Test the signing function for the eth_crypto."""
     account = EthereumCrypto(PRIVATE_KEY_PATH)
-    tx = Web3.solidityKeccak(['bytes'], [b'hello'])
+    tx = encode_defunct(text='hello')
```

Comment:
we might need to include encode_defunct in `sign_message` as a conversion step. Check if `tx_hash` remains the same if it is already bytes32